cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -795,12 +783,16 @@ func (c *client) kubectlDelete(manifest string) error {
 	return c.kubectlManifestCmd("delete", manifest)
 }
 
+func (c *client) kubectlDeleteAll(namespace, resource string) error {
```

Comment:
Do we need this function? It looks like it's a 1:1 wrap around `c.kubectlDeleteAllCmd`